swap/src/cli/command.rs

```diff
@@ -14,38 +18,219 @@ pub const DEFAULT_STAGENET_MONERO_DAEMON_ADDRESS_TESTNET: &str = "monero-stagene
 // See: https://1209k.com/bitcoin-eye/ele.php?chain=btc
 const DEFAULT_ELECTRUM_RPC_URL: &str = "ssl://electrum.blockstream.info:50002";
 // See: https://1209k.com/bitcoin-eye/ele.php?chain=tbtc
-const DEFAULT_ELECTRUM_RPC_URL_TESTNET: &str = "ssl://electrum.blockstream.info:60002";
+pub const DEFAULT_ELECTRUM_RPC_URL_TESTNET: &str = "ssl://electrum.blockstream.info:60002";
 
-const DEFAULT_BITCOIN_CONFIRMATION_TARGET: &str = "3";
-const DEFAULT_BITCOIN_CONFIRMATION_TARGET_TESTNET: &str = "1";
+const DEFAULT_BITCOIN_CONFIRMATION_TARGET: usize = 3;
+const DEFAULT_BITCOIN_CONFIRMATION_TARGET_TESTNET: usize = 3;
```

Comment:
I remember from our discussion from yesterday that you plan to align testnet and mainnet configs. 
I just want to point out that this is the target block, i.e. the fees will be estimated in a way so that your transaction will be included within 3 blocks. It does not say that we will be waiting for 3 confirmations.

Reply:
I understand that this parameter is different from the `finality_confirmations` and  that for testnet changing this value should not have any effect.
I aligned this param together with the other parameters so one does not have to think why this is different for `mainnet` and `testnet`. Unfortunately I did not align it for the ASB :/

I'll set it back to `1` for testnet - seems it is more logical afterall and the testnet/mainnet alignment caused more confusion than it helped in this case :)

Reply:
Yeah, totally aware that those are different values. After having a look at the commits: this is just a copy paster :D 
Changing it to `1` for testnet :)